src/game_legacy.h

```diff
@@ -142,9 +142,9 @@ struct Game {
     unsigned char eastegg02_cntr;
     char audiotrack;
     char last_audiotrack;
-char numfield_15;
+    char numfield_15;
```

Comment:
These missing indentations are there on purpose to make it easy to see the fields there still need to be named.